### ISSUE TITLE
Graphics.geometry.graphicsData not accessible in TypeScript

### ISSUE THREAD

**Issue by dirk-woehr** [pixijs/pixi.js#7343]
The graphicsData property of geometry is not accessible in TypeScript.

![Screenshot 2021-03-26 175358](https://user-images.githubusercontent.com/51853046/112666833-dc5ab680-8e5c-11eb-9a8e-e7bd329d4fb8.png)

**Comment by bigtimebuddy** [comment]
This works fine in 6.0.1, I assuming you mean 5.3.x?

**Comment by dirk-woehr** [comment]
Yup, you're right. I assumed we were already on 6.0.1 since I saw it in yarn info, but our package.json still was at 5.3.7. I updated that and now it works.

The use case is updating existing line widths after zooming, so they always are 1px on screen.

But after updating I now have another problem: VScode complains about geometry.invalidate() being protected. The build still works, but the red markers in VScode don't feel good ;) - Any Ideas how to bring VScode to accept that? Or is there 6.0.1 equivalent to geometry.invalidate()? Without calling it, the line widths don't get updated.

![Screenshot 2021-03-27 040637](https://user-images.githubusercontent.com/51853046/112708471-7436bf80-8eb2-11eb-8bda-37547c93768c.png)
